dk.bruchhaus.FFQueue.yml

```diff
@@ -0,0 +1,62 @@
+id: dk.bruchhaus.FFQueue
+runtime: org.gnome.Platform
+sdk: org.gnome.Sdk
+runtime-version: '49'
+#runtime: org.freedesktop.Platform
+#sdk: org.freedesktop.Sdk
+#runtime-version: '24.08'
+platform-extensions: 
+  - org.freedesktop.Platform.codecs-extra
+command: ffqueue
+rename-icon: ffqueue
+rename-desktop-file: ffqueue.desktop
+rename-appdata-file: ffqueue.xml
+
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --device=dri
+  - --share=ipc
+  - --filesystem=xdg-documents
+  - --filesystem=xdg-download
+  - --filesystem=xdg-music
+  - --filesystem=xdg-pictures
+  - --filesystem=xdg-videos
+  - --filesystem=/media
+  - --filesystem=/mnt
+
+cleanup:
+  - /lib/pkgconfig
+  - /bin/wx*
+  - /include
+  - /lib/debug/bin
+  - /lib/debug/lib
+  - /lib/debug/source
+  - /lib/wx
+
+modules:
+
+  - name: wxWidgets
+    buildsystem: autotools
+    config-opts:
+      - --enable-unicode
+      - --enable-shared
+      - --disable-debug
+      - --with-libpng
+      - --with-gtk=3
+    make-args:
+      - BUILD=release
+      - SHARED=1
+    sources:
+      - type: archive
+        url: https://github.com/wxWidgets/wxWidgets/releases/download/v3.2.8/wxWidgets-3.2.8.tar.bz2
+        sha256: c74784904109d7229e6894c85cfa068f1106a4a07c144afd78af41f373ee0fe6
+        
+  - name: FFQueue
+    buildsystem: autotools
+    sources:
+      - type: archive
+        url: https://github.com/bswebdk/FFQueue/archive/refs/tags/V1.7.70.tar.gz
+        sha256: 0cd76e24cd0d487053b0496d564d28eaebc5ee509e723886dab7c8f9b7547c2b
+      - type: shell
+        commands: [autoreconf -if]
```

Comment:
```suggestion
      - type: shell
        commands: 
          - autoreconf -if
```

This is yaml.

Reply:
are you gonna fight this also?
YAML syntax is a mistake. And your attitude perpetuate it.